### DISCUSSION TITLE
Redirect from / to locale path seems to ignore HTTP request language

### DISCUSSION THREAD

**Discussion by titus-kruse** [requarks/wiki#4582] [Help / Questions]
Hi! I'm new to wiki.js. I use Multilingual Namespacing and have configured two namespaces. German and English. Then I have a homepages with the paths /de/home and /en/home. But when I call the wiki in the browser with the root path /, it does not redirect to the correct language. HTTP request header Accept-Language is "de-DE,de;q=0.9,en-US;q=0.8,en;q=0.7". As HTTP response I get status 302 and location /en/home. Is this normal or am I doing something wrong? Current version 2.5.219.

**Comment by dbnicholson** [comment]
I don't believe you're doing anything wrong. Looking at the code that handles this, it doesn't look at `Accept-Language` at all and always redirects to the wiki's default language. It would be a nice feature to have it do a bit of content negotation there.

We ran into this recently where we want to provide a non-locale link to users and have it automatically redirect to the page matching the browser's configured language if it exists.
